config/redis.yml

```diff
@@ -69,12 +69,9 @@ development: &defaults
   pciu_address_dependencies:
     namespace: pciu-address-dependencies
     each_ttl: 604800
-  vet360_contact_info_response:
-    namespace: vet360-contact-info-response
+  va_profile_contact_info_response:
+    namespace: va-profile-contact-info-response
     each_ttl: 3600 # 1 hour
-  vet360_reference_data_response:
-    namespace: vet360-reference-data-response
-    each_ttl: 86400
```

Comment:
No rename on this one? What about the class that uses this key?

Reply:
Just grep'd and it looks like no classes use `vet360_reference_data_response`, 🤷  👍 .